pkg/newlog/cmd/newlogd.go

```diff
@@ -784,14 +804,40 @@ func checkAppEntry(entry *inputEntry) string {
 	return appuuid
 }
 
+func updateLogMsgID(domainName string) uint64 {
+	var msgid uint64
+	if domainName == "" {
+		msgid = msgIDDevCnt
+		msgIDDevCnt++
```

Comment:
Is this an error case? If you increment msgIDDevCnt but the log is sent to the app log stream there will be confusion around the skipped msgid for the device  log stream

Reply:
this is not an error case. all the msgID counter is handled in this function. the dev will not have uuid, thus the domainname is null.

Reply:
just added comments on this function for that

Reply:
OK